packages/lumx-react/src/components/thumbnail/Thumbnail.stories.tsx

```diff
@@ -20,6 +20,17 @@ export const defaultThumbnail = ({ theme }: { theme: Theme }) => {
         <Thumbnail
             align={select<Alignment>('Alignment', Alignment, Alignment.left, 'Options')}
             aspectRatio={select<AspectRatio>('Aspect ratio', AspectRatio, AspectRatio.square, 'Options')}
+            enableCrossOrigin={boolean('Enable CORS', true, 'Options')}
+            crossOrigin={select(
+                'CORS',
+                {
+                    anonymous: CrossOrigin.anonymous,
+                    useCredentials: CrossOrigin.useCredentials,
+                },
```

Comment:
```suggestion
               CrossOrigin,
```
Using the `enum` directly also works!